extension-manifest-v3/xcode/iOS (App)/Info.plist

```diff
@@ -23,7 +23,5 @@
 			</array>
 		</dict>
 	</dict>
-	<key>UIStatusBarStyle</key>
```

Comment:
Not sure why, but it was automatically removed by the Xcode, not by hand. I checked, and the App UI still looks like it should.